### DISCUSSION TITLE
MSG: "Your ComfyUI is too OUTDATED!!!"

### DISCUSSION THREAD

**Discussion by ghost** [Comfy-Org/ComfyUI-Manager#830] [General]
I downloaded and reinstalled Comfy today. In the Manager I got the message  "Your ComfyUI is too OUTDATED!!!" (it shows ComfyUI: Unknown[-](1900-01-01), Manager: V2.43). When updating in manager I get "Failed to update ComfyUI." (Update ComfyUI
ComfyUI update fail: The installed ComfyUI does not have a Git repository.)

BTW: Thanks for this amazing tool!!!!!

**Comment by ltdrdata** [comment]
This is answered already.
